app/assets/javascripts/preview.js

```diff
@@ -0,0 +1,5 @@
+$(document).ready(function() {
+
+  $('.header-preview').toggle();
```

Comment:
Rather than doing this with javascript on load, it would be better to just put in the css:

```css
.header-preview {
  display: none;
}
```